### PR TITLE
Issue #SB-29126 fix: Contribute to RC on QR code generation with specific type

### PR DESCRIPTION
**Pull request by VenkatKadiveti**

### CODE REVIEW

File: services/certificate-api/src/routes/certificate_controller.js [hunk 1]
```diff
@@ -97,27 +99,32 @@ const getRequestBody = async (req) => {
...
+    const zippedData = await zip.generateAsync({type: zipType ? 'base64': 'string', compression: "DEFLATE"})
         .then(function (content) {
             // console.log(content)
             return content;
         });
-
-    const dataURL = await QRCode.toDataURL(zippedData, {scale: 3});
+    let qrData = zippedData;
+    if (zipType) {
+        qrData = `${envData.certDomainUrl}/certs/${entityId}?t=${envData.qrType}&&data=${zippedData}`;
```
Comment (tejash-jl) [inline services/certificate-api/src/routes/certificate_controller.js:118]: **&&** is it added intentionally?

File: services/certificate-api/src/routes/certificate_controller.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ const puppeteer = require('puppeteer');
 const QRCode = require('qrcode');
 const JSZip = require("jszip");
 const { default: axios } = require('axios');
+const QRTYPE = 'URL-W3C-VC';
```
Comment (tejash-jl) [inline services/certificate-api/src/routes/certificate_controller.js:8]: rename the const name to URL_BASED_QR or something better?

File: java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java [hunk 3]
```diff
@@ -450,6 +450,7 @@ private ObjectNode copyWhiteListedFields(ArrayList<String> fields, JsonNode data
             node = objectMapper.readTree(node.get(OSSystemFields._osSignedData.name()).asText());
             return new ResponseEntity<>(certificateService.getCertificate(node,
                     entityName,
+                    entityId,
```
Comment (tejash-jl) [inline java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java:453]: the node object will already contain entityId or Osid,
do we still need to pass it externally?

File: services/certificate-api/src/routes/certificate_controller.js (hunk 2 above)
Comment (VenkatKadiveti) [reply services/certificate-api/src/routes/certificate_controller.js:8]: ok

File: services/certificate-api/src/routes/certificate_controller.js (hunk 1 above)
Comment (VenkatKadiveti) [reply services/certificate-api/src/routes/certificate_controller.js:118]: yes

**Review by tejash-jl — APPROVED** [review]

**Pull request merged**